prototype/link/scripts/retinotopy/README.md

```diff
@@ -0,0 +1,138 @@
+# Analysis pipeline for retinotopy
+
+This outlines the analysis pipeline for running retinotopy analyses. This assumes that the normal infant\_neuropipe steps have been completed.
+
+1. Run retinotopy GLMs.
+Use **sbatch ./scripts/retinotopy/supervisor_retinotopy.sh** to run the necessary GLMs to compare high and low spatial frequency, horizontal and vertical meridians, and the F-test of all conditions.
+
+
+2. Create a Freesurfer directory that you can use for surface reconstruction.
+This directory can be created with FreeSurfer or by adapting iBEAT. To use Freesurfer you can use `scripts/run_recon.sh` as a script for launching the job. If using iBEAT then follow the steps described in the `convert_iBEAT-FS.md` README. This freesurfer directory must be able to have SUMA created from it.
+
+
+3. Make the cuts in the inflated surface using tksurfer
+With the freesurfer directory set up you can run tksurfer to make cuts of the inflated surface. Where you should make the cuts is outlined here: https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferOccipitalFlattenedPatch
+
+It might be possible to run this on the cluster but you probably want to pull it locally to keep your sanity. Regardless I am assuming that you are in a subject directory and that there is a folder called `./analysis/freesurfer/iBEAT/`. Once in the participant directory run these commands, once for each hemisphere. This command creates a gifti file that can be used to help guide the cut, and then loads it in to tksurfer. It assumes you have a file called `meridian_zstat3_aligned.nii.gz` in the director which is created by the GLM in step (1) or alternatively can be found in the data release in the `contrast_maps` folder:
```

Comment:
It was a confusing section so I have reworded. This shouldn't loop through though, since tksurfer is a gui that you need to interact with, as the preceding paragraph explains